src/components/exercises/details.cjsx

```diff
@@ -0,0 +1,151 @@
+React     = require 'react'
+BS        = require 'react-bootstrap'
+keymaster = require 'keymaster'
+
+{ExerciseStore}   = require '../../flux/exercise'
+{ExercisePreview} = require 'openstax-react-components'
+NoExercisesFound  = require './no-exercises-found'
+Icon              = require '../icon'
+ScrollTo          = require '../scroll-to-mixin'
+
+KEYBINDING_SCOPE  = 'exercise-details'
+
+ExerciseDetails = React.createClass
+
+  propTypes:
+    selectedExercise:      React.PropTypes.object.isRequired
+    selectedSection:       React.PropTypes.string
+    exercises:             React.PropTypes.object.isRequired
+    onSectionChange:       React.PropTypes.func
+    onExerciseToggle:      React.PropTypes.func.isRequired
+    onShowCardViewClick:   React.PropTypes.func.isRequired
+    topScrollOffset:       React.PropTypes.number
+    getExerciseActions:    React.PropTypes.func.isRequired
+    getExerciseIsSelected: React.PropTypes.func.isRequired
+
+  mixins: [ScrollTo]
+  scrollingTargetDOM: -> document
+  getDefaultProps: ->
+    topScrollOffset: 40
```

Comment:
Can you add a comment on why this is 40px or if it's used elsewhere can it be a const that is pulled in to this file?